src/App.js

```diff
@@ -1,10 +1,19 @@
-
-import './App.css';
+import { useState } from "react";
+import { BrowserRouter } from "react-router-dom";
+import { UserContext } from "./components/User";
+import "./App.css";
+import Container from "./components/Container";
 
 function App() {
+  const [user, setUser] = useState("grumpy19");
```

Comment:
I think you might be jumping the gun a bit here, you wont need this until later :)